### DISCUSSION TITLE
Ways of ensuring that answers can be edited afterwards

### DISCUSSION THREAD

**Discussion by mpldr** [lukevella/rallly#1032] [Ideas]
It has been brought up by users that they were unable to update their answers afterwards. Whether they tried using the same browser as the original filling out, I can not say. I suggest adding a link to the confirmation mail that allows the user to edit their responses from every device. Something like:

> If you wish to edit your responses, you can do so at https://rallly.my-doma.in/edit/lolololo

**Comment by lukevella** [comment]
This is already how it works. If participants type in their email address they will receive a confirmation email with a link to edit their response from any device. The link doesn't expire.

**Reply by mpldr** [reply]
understood, I shall blame the user ^^
